### PR TITLE
Make the directories' names look better

### PR DESCRIPTION
**Pull request by ghost**
Make the directories' names look better
Added description for "Install in VS 2019"

### CODE REVIEW

File: nsis/visuald.nsi [hunk 1]
```diff
@@ -27,7 +27,7 @@
 ; define DMD source path to include dmd installation
 ; !define DMD 
 !define DMD_VERSION "2.086.0"
-!define DMD_SRC c:\l\d\dmd${DMD_VERSION}\dmd2
+!define DMD_SRC c:\l\d\dmd-${DMD_VERSION}\dmd2
```
Comment (rainers) [inline nsis/visuald.nsi]: That's the source directory on my disk used when the installer is built, so no use in changing it.

File: nsis/visuald.nsi [hunk 2]
```diff
@@ -90,9 +90,9 @@
   Caption "${LONG_APPNAME} ${VERSION} Setup"
 !ifdef DMD
   !ifdef LDC
-    !define OUT_SUFFIX "-dmd${DMD_VERSION}-ldc2-${LDC_VERSION}"
+    !define OUT_SUFFIX "-dmd-${DMD_VERSION}-ldc2-${LDC_VERSION}"
   !else
-    !define OUT_SUFFIX "-dmd${DMD_VERSION}"
+    !define OUT_SUFFIX "-dmd-${DMD_VERSION}"
```
Comment (rainers) [inline nsis/visuald.nsi:95]: These could indeed change for more consistency with ldc.

File: nsis/visuald.nsi [hunk 3]
```diff
@@ -330,7 +330,7 @@ ${MementoSection} "Install DMD" SecDMD
 
   ; not using ${File} to keep compiler even if Visual D uninstalled
 
-  !define DmdBaseDir "$CompilerInstallDir\dmd${DMD_VERSION}"
+  !define DmdBaseDir "$CompilerInstallDir\dmd-${DMD_VERSION}"
```
Comment (rainers) [inline nsis/visuald.nsi:333]: This name is also used in the update process, so changes would have to be made in visuald/updates.d, too. I think it fits with the default D installation that uses `dmd2` in the same folder, though.

**Review by rainers — CHANGES_REQUESTED** [review]
Thanks for the contribution. See comments for the two names that should be kept, though.

**Comment by ghost** [comment]
Changes made.

File: nsis/visuald.nsi (hunk 3 above)
Comment (ghost) [reply nsis/visuald.nsi:333]: It's ugly to append a "full" version number to the product name directly, so in most cases there would be a hyphen or underline to separate them, such as the name of dmd installer executive.

**Comment by rainers** [comment]
Thanks!

**Pull request merged**